packages/button/src/css/index.js

```diff
@@ -50,7 +50,14 @@ export default {
       }
     }
   },
-
+  '.psds-button--psds2020Colors': {
```

Comment:
a couple naming options that might be cleaner and follow BEM standards

`psds-button--2020-colors`
`psds-button--feature-2020-colors`

🤷‍♂

Reply:
Maybe will have to refactor a lot of other commit for that to work.